driver/razerkbd_driver.c

```diff
@@ -1687,6 +1696,11 @@ static ssize_t razer_attr_write_matrix_effect_wheel(struct device *dev, struct d
     struct razer_report response = {0};
 
     switch(device->usb_pid) {
+    case USB_DEVICE_ID_RAZER_BLACKWIDOW_V4:
+        request = razer_chroma_extended_matrix_effect_wheel(VARSTORE, BACKLIGHT_LED, direction);
+        request.transaction_id.id = 0x1F;
+        break;
+
```

Comment:
Why are you adding the code again? Just re-use the case for V4_PRO?

Reply:
Ok, fixed all V4/V4_PRO duplicates.